dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/AclEventExporterTest.java

```diff
@@ -369,11 +373,37 @@ void shouldReturnSelectedOrgUnitEventsWhenNoOuModeSpecifiedAndUserHasAccessToOrg
                     + " instead"));
   }
 
+  @Test
+  void shouldReturnAccessibleOrgUnitEventsWhenNoOuModeNorOrgUnitSpecifiedAndUserHasAccessToOrgUnit()
```

Comment:
Can this name be simplified to

```suggestion
  void shouldReturnAccessibleOrgUnitEvents()
```

the exact params are visible from the builder

Reply:
I guess that's just my personal taste, but I like having the important details of the test in the title, so just by reading it I know what to expect.

Reply:
Yes probably subjective but to me the essence is getting the accessible org units 😅  Reapeating most params seems to redundant/verbose IMHO